msteams-platform/concepts/tabs/tabs-requirements.md

```diff
@@ -15,30 +14,29 @@ All tab content, including configuration, content, and tab-removal pages must me
+
 Tabs behave differently on the iOS and Android Teams clients than on the Teams desktop or web clients.
 
 ### Desktop and web client behavior
 
-What appears in a tab on desktop/web is defined by the value of `contentUrl`: Teams displays the contents of `contentUrl` in the tab. (See [Determine the content to display in the tab](~/concepts/tabs/tabs-configuration#determine-the-content-to-display-in-the-tab)). If the tab has a value for `websiteUrl`, another icon appears with the others at the upper right of the tab: ![Go to website](~/assets/images/go-to-website-icon.png). When the user clicks on this icon, `websiteUrl` is opened in a new browser tab.
+What appears in a tab on desktop/web is defined by the value of `contentUrl`: Teams displays the contents of `contentUrl` in the tab. (See [Determine the content to display in the tab](~/concepts/tabs/tabs-configuration.md#determine-the-content-to-display-in-the-tab)). If the tab has a value for `websiteUrl`, another icon appears with the others at the upper right of the tab: ![Go to website](~/assets/images/go-to-website-icon.png). When the user clicks on this icon, `websiteUrl` is opened in a new browser tab.
 
 ### Mobile client behavior
 
-On mobile clients, where there's far less screen real estate, it's not possible to show the contents of files and tabs embedded inside Teams; instead, the user navigates to them via the "Files & Tabs" tab. The mobile clients only show tabs that have a value for `websiteUrl`: if you want your tab to appear on the Teams mobile clients, you must set the value of `websiteUrl`.
-
-Since the Teams mobile clients use the mobile OS to open `websiteUrl`, you can use iOS *universal links* or Android *app links* to open `websiteUrl` in your mobile app. The OS will also prompt the user to install your app if it is not already installed.
+> [!Important]
+> Full support for tabs on mobile clients is coming soon. To prepare for this change you should follow the [guidance for personal apps on mobile](~/resources/design/framework/tabs-mobile.md) when creating your tabs. When full support for tabs is release, your `contentUrl` **will be loaded in the mobile Teams client**. Your users will still have the option of opening your tab in a separate browser via your `websiteUrl`, however your `contentUrl` will be loaded first.
 
-Information on universal links and app links:
-
-* [iOS universal links](https://developer.apple.com/ios/universal-links/)
-* [Android app links](https://developer.android.com/training/app-links/index.html)
+On mobile clients, where there's far less screen real estate, it's not possible to show the contents of files and tabs embedded inside Teams; instead, the user navigates to them via the "Files & Tabs" tab. The mobile clients only show tabs that have a value for `websiteUrl`: if you want your tab to appear on the Teams mobile clients, you must set the value of `websiteUrl`.
```

Comment:
Suggesting to change this to:

Today on mobile clients, user can view the list of configurable tabs for a channel by navigating to "More" tab. The mobile clients only show tabs that have a value for `websiteUrl`: if you want your tab to appear on the Teams mobile clients, you must set the value of `websiteUrl`. However when full support for static tabs on mobile clients is released, 
along with that we will start showing all configurable tabs ( Even if 'websiteUrl' is not available for a tab)

If user opens a configurable tab on browser using 'websiteUrl', you can use iOS *universal links* or Android *app links* to deeplink `websiteUrl` in your mobile app if you want to. The OS will also prompt the user to install your app if it is not already installed.

Information on universal links and app links:
iOS universal links
Android app links

Reply:
Changed your suggestion a bit, and updated. The text outside of the of the note should be what is generally applicable today. Also took this expanded notice and added it to the tabs overview page.